.github/workflows/ci.yml

```diff
@@ -10,17 +10,13 @@ on:
 
 jobs:
 
-    tests:
-        name: Tests - PHP ${{ matrix.php }} ${{ matrix.dependency-version }}
+    tests-ubuntu:
+        name: Tests Ubuntu - PHP ${{ matrix.php }} ${{ matrix.dependency-version }}
         runs-on: ubuntu-latest
         timeout-minutes: 15
         strategy:
             matrix:
                 php: [ '8.0', '8.1', '8.2', '8.3' ]
```

Comment:
try adding `os: [ubuntu-latest, windows-latest]` to the matrix, instead of duplicating all steps for windows

Reply:
https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs

Reply:
@falkenhawk thanks, the initial division was in case the setup of PHP for windows is different, but it turned out to be the same so you are right :)